.github/workflows/full-stack.yaml

```diff
@@ -95,13 +96,17 @@ jobs:
           openff-qcsubmit/ \
           openff-units/ \
           openff-models/ \
+          openff-fragmenter/ \
           smirnoff-plugins/
 
     - name: Run 'everything all at once' tests
       run: |
         python -m pytest $PYTEST_ARGS \
           smirnoff-plugins/smirnoff_plugins/_tests/ \
           openff-interchange/openff/interchange/_tests/ \
+          openff-models/openff/models/_tests/ \
+          openff-units/openff/units/_tests/ \
```

Comment:
This is largely motivated by an attempt to ensure https://github.com/openforcefield/openff-units/pull/84 doesn't cause any downstream effects we haven't already detected